### PR TITLE
Add my (Chris Knepper) YAML file for Participants page

### PR DESCRIPTION
**Pull request by chrisknepper**

### CODE REVIEW

File: scripts/people/ChrisKnepper.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Hi!  I am threebean
```
Comment (decause) [inline scripts/people/ChrisKnepper.yaml]: orlly? Threebean you say? I know another Threebean too... 
:wink:

File: scripts/people/ChrisKnepper.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+- blog: http://beta.chrisknepper.com/blog
+  feed: http://beta.chrisknepper.com/feed
+  forges:
+    - http://github.com/ralphbean
```
Comment (decause) [inline scripts/people/ChrisKnepper.yaml]: :arrow_double_up: :question:

**Comment by chrisknepper** [comment]
I fixed it yay!

File: scripts/people/ChrisKnepper.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Hi!  I am ChrisKnepper
+- blog: http://beta.chrisknepper.com/blog
+  feed: http://beta.chrisknepper.com/feed
+  forges:
+    - http://github.com/ralphbean
```
Comment (decause) [inline scripts/people/ChrisKnepper.yaml]: :question: Still wrong forge link I'm afraid :/

**Comment by chrisknepper** [comment]
I feel really dumb ;_;

**Comment by decause** [comment]
That's much better.
:+1:

**Pull request merged**